### ISSUE TITLE
2.9.4dev2 throws an error on boot

### ISSUE THREAD

**Issue by MarHelmer** [pyfa-org/Pyfa#1971]
## Bug Report
```
pyfa v2.9.4dev2
EVE Data Version: 1503340 (2019-05-09 09:14:26)

OS version: Windows-10-10.0.17763-SP0
Python version: 3.6.8 (tags/v3.6.8:3c6b436a57, Dec 23 2018, 23:31:17) [MSC v.1916 32 bit (Intel)]
wxPython version: 4.0.4 (wxWidgets 3.0.5)
SQLAlchemy version: 1.3.3
Logbook version: 1.4.3
Requests version: 2.21.0
Dateutil version: 2.8.0

####################

Traceback (most recent call last):
  File "C:\projects\pyfa\pyfa.py", line 132, in <module>
  File "<frozen importlib._bootstrap>", line 971, in _find_and_load
  File "<frozen importlib._bootstrap>", line 955, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 665, in _load_unlocked
  File "C:\Python36\lib\site-packages\PyInstaller\loader\pyimod03_importers.py", line 631, in exec_module
  File "C:\projects\pyfa\gui\mainFrame.py", line 41, in <module>
  File "<frozen importlib._bootstrap>", line 971, in _find_and_load
  File "<frozen importlib._bootstrap>", line 955, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 665, in _load_unlocked
  File "C:\Python36\lib\site-packages\PyInstaller\loader\pyimod03_importers.py", line 631, in exec_module
  File "C:\projects\pyfa\gui\graphFrame.py", line 31, in <module>
  File "<frozen importlib._bootstrap>", line 971, in _find_and_load
  File "<frozen importlib._bootstrap>", line 955, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 665, in _load_unlocked
  File "C:\Python36\lib\site-packages\PyInstaller\loader\pyimod03_importers.py", line 631, in exec_module
  File "C:\projects\pyfa\gui\graph.py", line 43, in <module>
AttributeError: module 'gui.builtinGraphs' has no attribute 'fitDpsRange'
```

### Expected behavior:
Pyfa should boot normally.

### Actual behavior:
Pyfa does not boot.  The above error message displays, and the task does not close.  The crashed instance of Pyfa must be manually closed through the Task Manager.

### Detailed steps to reproduce:
Install 2.9.4dev2, then try to boot Pyfa.

### Fits involved in EFT format (Edit > To Clipboard > EFT):
N/A

### Release or development git branch? Please note the release version or commit hash:
2.9.4dev2

### Operating system and version (eg: Windows 10, OS X 10.9, OS X 10.11, Ubuntu 16.10):
Windows 10

### Other relevant information:
This has persisted through multiple reinstalls and was not present in previous releases.

**Comment by DarkFenX** [comment]
Looks like pyinstaller doesn't pick up this line: https://github.com/pyfa-org/Pyfa/blob/singularity/gui/graph.py#L43

**Comment by DarkFenX** [comment]
Fixed in v2.9.4dev3
